src/ai/backend/client/cli/vfolder.py

```diff
@@ -90,7 +90,11 @@ def list_allowed_types():
               help='Folder\'s innate permission. '
                    'Group folders can be shared as read-only by setting this option to "ro".'
                    'Invited folders override this setting by its own invitation permission.')
-def create(name, host, group, host_path, usage_mode, permission):
+@click.option('-q', '--quota', metavar='QUOTA', type=str, default=None,
+              help='Quota of the virtual folder. '
+                   '(Use \'m\' for megabytes, \'g\' for gigabytes, and etc.) '
+                   'Default is maximum amount possible.')
```

Comment:
💡 Tip: Use `ai.backend.client.cli.utils.ByteSizeParamType` here. You may also define a subclass of it which does *not* convert the expression into integers but only validates the binary scale suffix.